internal/nodejs-action/generate-layout.sh

```diff
@@ -20,17 +48,18 @@ alg=$(echo "${integrity_digest}" | cut -d'-' -f1 | tr '[:upper:]' '[:lower:]')
 digest=$(echo "${integrity_digest}" | cut -d'-' -f2- | base64 -d | od -A n -v -t x1 | tr -d ' \n')
 
 # NOTE: the name of the attestation should be configurable.
+attestation_name="attestation.intoto"
```

Comment:
what if it's just the packages `<name>.intoto`?

Reply:
I updated this to use the package `filename` included in the output of `npm pack --json`. I can't just use the package name since it could be something like `@ianlewis/mypackage` if it's a scoped package.

Right now though, this is only used internally in the workflow as the provenance is uploaded as json metadata by `npm publish` so this file isn't actually stored anywhere.